1Password/shell-plugins
# [BUG FIX] Remove ngrok version check everytime a op plugin command is called

## Overview


This MR will fix a bug regarding `ngrok --version` being called on every `op plugin` command.

## Type of change
 


## Related Issue(s)
  

* Resolves: #
* Relates: #

## How To Test

The test pipelines should pass as usual and when running an `op plugin` command, if you check the activity monitor.. there should be no `ngrok` processes. (Note: Must have the ngrok CLI installed to test this)

Sometimes its hard to see the `ngrok` process on the activity monitor/task manager. An easier way to test is by changing L129 on `plugins/ngrok/provisioner.go` to open up any application you have on your PC and see if the app opens up when you run a `op plugin` command.

## Changelog
  
The op plugin commands do not call a ngrok version check even if the ngrok plugin is not initialized.
